tests/integration/test_tiler.py

```diff
@@ -18,13 +18,32 @@
 
 class DescribeRandomTiler:
     @pytest.mark.parametrize(
-        "fixture_slide, binary_mask, tile_size, level, check_tissue, expectation",
+        "fixture_slide, binary_mask, tile_size, level, outline, check_tissue, expectation",
```

Comment:
this line is 91 char. Be sure you have black formatter before commit

Reply:
@ernestoarbitrio Actually I did use black .. it automatically runs on all my commits with the precommit-hooks. I ran it again now and it didn't make any changes to that file. Are you sure 91 char is the maximum allowed?

Reply:
88 is the max

Reply:
```
.\tests\integration\test_tiler.py:21:89: E501 line too long (91 > 88 characters)
```

Reply:
black doesn't take care of single strings > than 88 chars

Reply:
OK, fixed.